meta-phosphor/classes/obmc-phosphor-image.bbclass

```diff
@@ -48,9 +48,12 @@ CORE_IMAGE_EXTRA_INSTALL_append = " bash \
 
 OBMC_IMAGE_EXTRA_INSTALL ?= ""
 
-def build_overlay(d):
-        if bb.utils.contains("IMAGE_FSTYPES", "overlay", "overlay", "0", d) != "0":
-                return "image-overlay"
-        return ""
-
-inherit ${@build_overlay(d)}
+def image_overlay_enabled(d, fstype):
+        if d.getVar('OBMC_PHOSPHOR_IMAGE_OVERLAY', True) != "1":
+            return ""
+        if fstype:
+            return "overlay"
+        return "image-overlay"
+
+IMAGE_FSTYPES += "${@image_overlay_enabled(d, True)}"
```

Comment:
Using the 2nd arg to toggle between fixed use cases (IMAGE_FSTYPES and class to inherit from) is ugly.  Maybe take a string that is used as the value to return on match similar to bb.utils.contains()?